charter.md

```diff
@@ -23,48 +23,33 @@ The WebTransport working group will define new client-server protocols
 or protocol extensions in order to support the development of the
 W3C WebTransport API <https://wicg.github.io/web-transport>.
 
-These protocols will support:
-  * Reliable bidirectional and unidirectional communication
-    that provides greater efficiency than Websockets
-    (e.g. removal of head-of-line blocking).
-  * Unreliable datagram communication, functionality not available
-    in Websockets.
-  * Origin checks to allow supporting the Web's origin-based
-    security model.
-
-The WebTransport working group will define three variants:
-  * A protocol directly running over QUIC with its own ALPN.
-  * A protocol that runs multiplexed with HTTP/3.
-  * Fallback protocols that can be used when QUIC or UDP 
-    are not available.
+The WebTransport working group will define a protocol or suite of
+protocols that support a range of simple communication methods.
+These must include unreliable messages (that might be limited by
+the path MTU), reliable messages, and ordered streams of reliable
+messages.  Attention will be paid to the performance of the protocol,
```

Comment:
Is the use case of a byte stream captured by defining a message with a length of 1 byte?  I'd prefer some kind of description of that use case, since I expect it to be fairly common.

Reply:
Or, conversely, a stream is a really long message that's over when the stream is closed.

Reply:
I think this is so technically detailed that it doesn't need to be spelled out in the charter, we can discuss that in the documents themselves.

Reply:
I don't find it to be completely in the weeds.  That said, as long as we're going to support a byte stream use case,  and no one is going to push back saying "the charter only says messages", then I'm OK with this.